src/style.css

```diff
@@ -1 +1,30 @@
 /* add styles here */
+* {
```

Comment:
Don't use * selector (it impacts performance)
Better use ``body`` for this

![image](https://github.com/mate-academy/layout_stars/assets/54860641/1f0f4c2f-d183-4d0d-9a24-ee105d695bcc)